apps/admin-ui/src/realm-settings/NewClientPolicyForm.tsx

```diff
@@ -401,7 +401,7 @@ export default function NewClientPolicyForm() {
       (policy) => createdPolicy.name === policy.name
     );
 
-    if (!index || index === -1) {
+    if (index === undefined || index === -1) {
```

Comment:
Perhaps this is something we could lint? Ideally we'd never use these kind of conditions on numbers, as they always could be `0`.

Sidenote, would it perhaps be better to assert the type of `number` here instead?
```suggestion
    if (typeof index !=='number' || index === -1) {
```